`filter_spatial`: Clarify masking (#470)
Open-EO/openeo-processes: CHANGELOG.md

## Unreleased / Draft

### Fixed

> - `filter_spatial`: Clarified that a masking get applied for the given geometries. [#469](https://github.com/Open-EO/openeo-processes/issues/469)

Reviewer: ```suggestion
- `filter_spatial`: Clarified that masking is applied using the given geometries. [#469](https://github.com/Open-EO/openeo-processes/issues/469)
```
